docs/Users_Guide/release-notes.rst

```diff
@@ -22,23 +27,35 @@ METplotpy Version 3.1.0-beta2 release notes (20250331)
    * Plotly line plots are plotting confidence limit bars at zero with zero length error bars (`#495 <https://github.com/dtcenter/METplotpy/issues/495>`_)
 
```

Comment:
These additional bugfixes used to be listed here, but I see they are no longer.  Did you mean to include them?
```
   * Series names in legend don't get ordered correctly  (`#347 <https://github.com/dtcenter/METplotpy/issues/347>`_)
   * Support various formats of show_legend values (`#482 <https://github.com/dtcenter/METplotpy/issues/482>`_)
   * Reliability Diagram show_legend setting should work with True/False rather than 1/0 values; see bugfix #482(`#455 <https://github.com/dtcenter/METplotpy/issues/455>`_)
   * Better handling of determining min and max for confidence limits when data contains NaN values(`#494 <https://github.com/dtcenter/METplotpy/issues/494>`_)
```

Reply:
they are included